lib/modules/manager/composer/extract.spec.ts

```diff
@@ -10,6 +10,7 @@ const requirements3 = Fixtures.get('composer3.json');
 const requirements4 = Fixtures.get('composer4.json');
 const requirements5 = Fixtures.get('composer5.json');
 const requirements5Lock = Fixtures.get('composer5.lock');
+const requirements6 = Fixtures.get('composer6.json');
```

Comment:
Please inline this file using `codeBlock` syntax so that we don't need to add another file to be read from `fs`